src/state/login-form/actions.js

```diff
@@ -34,8 +34,14 @@ export const performLogin = (username, password) => dispatch => (
           dispatch(setLoginErrorMessage({ id: 'fcc.login.errorMessage', defaultMessage: ERROR_LOGIN_MESSAGE }));
           resolve();
         }
-      }).catch(() => {
-        dispatch(setLoginErrorMessage({ id: 'fcc.login.errorMessage', defaultMessage: ERROR_LOGIN_MESSAGE }));
+      }).catch((err) => {
```

Comment:
What about having an error handling function that chooses the proper message to display, so the action calls it and the `LoginFormContainer` simply displays the processed message?